code/modules/mining/ores_coins.dm

```diff
@@ -159,55 +149,60 @@
 
 /obj/item/stack/ore/mauxite
 	name = "mauxite ore"
-	desc = "A chunk of Mauxite, a sturdy common metal."
+	desc = "A chunk of Mauxite, a sturdy ore used on Clown Planet."
 	icon_state = "mauxite"
-	material="mauxite"
+	starting_materials = list(MAT_IRON = CC_PER_SHEET_METAL*0.9, MAT_CLOWN = CC_PER_SHEET_CLOWN*0.1)
```

Comment:
Making these goon ores use regular materials is an interesting choice but also makes them kinda useless I feel.

They could still be used for some niche metallurgy use, at least some of them.